Airports/EGOV/Active_Runway.txt

```diff
@@ -1,2 +1,4 @@
 ACTIVE_RUNWAY:EGOV:31:1
-ACTIVE_RUNWAY:EGOV:31:0
\ No newline at end of file
+ACTIVE_RUNWAY:EGOV:31:0
+ACTIVE_RUNWAY:EGOV:01:0
+ACTIVE_RUNWAY:EGOV:01:0
```

Comment:
This sets the default arrival and departure runways in the Euroscope runway setting dialog. 

If you want 31 to be the default arrival/departure runway what you want is:
```suggestion
ACTIVE_RUNWAY:EGOV:31:0
```